tests/MediaTest.php

```diff
@@ -0,0 +1,332 @@
+<?php
+
+namespace RonasIT\Media\Tests;
+
+use App\Models\User;
+use RonasIT\Media\Models\Media;
+use App\Tests\Support\MediaTestTrait;
+use Illuminate\Http\Testing\File;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use RonasIT\Support\Traits\FilesUploadTrait;
+
+class MediaTest extends ModuleTestCase
+{
+    use FilesUploadTrait;
+    use MediaTestTrait;
+
+    protected $admin;
+    protected $user;
+    protected File $file;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->admin = User::find(1);
```

Comment:
please implement static logic here

```suggestion
    protected static $admin;

    public function setUp(): void
    {
        parent::setUp();

        self::$admin ??= User::find(1);
```

Reply:
Deleted the tests because it was decided to do them in another merge request.